TombEngine/Game/Lara/lara_fire.cpp

```diff
@@ -786,10 +786,10 @@ FireWeaponType FireWeapon(LaraWeaponType weaponType, ItemInfo& targetEntity, Ite
 	auto& player = *GetLaraInfo(&laraItem);
 	auto& ammo = GetAmmo(player, weaponType);
 
-	if (ammo.GetCount() == 0 && !ammo.HasInfinite())
+	if (ammo.GetCount() == 0 && !ammo.HasInfinite() && weaponType != LaraWeaponType::Snowmobile)
```

Comment:
I think that maybe better solution will be assigning ammo to snowmobile as infinite? This looks like a hack.